src/pages/calendar.tsx

```diff
@@ -1,11 +1,11 @@
 import CalendarPage from "@/components/Calendar";
 import { getAuthenticatedUserFromSession } from "@/utils/passage";
 import { getSupabase } from "@/utils/supabase";
-import { AuthProps, ComponentProps } from "@/types/types";
+import { AuthProps } from "@/types/types";
 import { GetServerSideProps } from "next";
 
-export default function calendar({isAuthorized, logOut, data}: ComponentProps) {
-  return (<CalendarPage isAuthorized={isAuthorized} logOut={logOut} data={data} />)
+export default function calendar({isAuthorized}: AuthProps) {
+  return (<CalendarPage isAuthorized={isAuthorized} userID={""} />)
```

Comment:
maybe  I am just confused on how this works but is there a reason userID is an empty string? Should we change whether or not we need it in the types?

Reply:
oops I think this was a mistake 😂